include/oneapi/tbb/parallel_for.h

```diff
@@ -319,7 +319,7 @@ void parallel_for_impl(Index first, Index last, Index step, const Function& f, P
 template <typename Index, typename Function>
     __TBB_requires(parallel_for_index<Index> && parallel_for_function<Function, Index>)
 void parallel_for(Index first, Index last, Index step, const Function& f) {
-    parallel_for_impl<Index,Function,const auto_partitioner>(first, last, step, f, auto_partitioner());
+    parallel_for_impl<Index,Function,const __TBB_DEFAULT_PARTITIONER>(first, last, step, f, __TBB_DEFAULT_PARTITIONER());
```

Comment:
is `const` redundant here? in case `__TBB_DEFAULT_PARTITIONER` is `affinity_partitioner` for example

Reply:
nvm, it's ok for `affinity_partitioner` to be `const`, anyway `affinity_partitioner::task_partition_type` is used for actual partitioning